packages/liveblocks-react-ui/src/primitives/index.ts

```diff
@@ -1,5 +1,7 @@
 export type { ComposerBodyMark, ComposerBodyMarks } from "../types";
 export * as AiChatComposer from "./AiChatComposer";
+// XXX Move into /internal instead of exposing it as a Primitive here right now?
+export * as AiMessage from "./AiMessage";
```

Comment:
Any preferences for where to export this? It's a primitive, but also a private API, so I couldn't decide. cc @marcbouchenoire @nimeshnayaju

Reply:
I personally prefer from `_private` like we do for the `AiChatComposer`, the distinction between `@liveblocks/react-ui` and `@liveblocks/react-ui/primitives` is only to signal to users what kind of components they are importing (especially when we have a `Comment` **primitive** and a `Comment` **default component**), but imo we ourselves don't need it.

Reply:
Same! It's a private API for now, so I'd keep it in `_private`. The goal is to move it to `primitives` once it's public.